### ISSUE TITLE
[Bug] BetterDiscord wont open during startup

### ISSUE THREAD

**Issue by Waryth** [BetterDiscord/BetterDiscord#1067]
**Describe the Bug**
I have **Open Discord**, **Start Minimized** and **Minimized to Tray** enabled in my Discord's Window Settings. Whenever I start my PC and all my startup applications such as Steam, Anti-Virus and everything else opens up except Discord. I wait up til 5 minutes long to even confirm it.

So what I did was tried fixing it by repairing it using the installer and reinstalled it but didn't fixed it. I decided to remove the plugins by renaming the plugins folder so there's no plugins running with it and restarted. It doesn't start still during startup.

Yes, I checked my Windows tray at my taskbar to see if Discord is just silently hiding and there's none. Everytime I manually open Discord it boots up the loading screen instead of just opening up so it really didn't started.

**To Reproduce**
1. Go to user settings
2. Click on Windows Settings which is in **APP SETTINGS** category
3. Enable **Open Discord**, **Start Minimized**, **Minimized to Tray**. All those 3 must be enabled.
4. Restart your PC
5. Discord with BetterDiscord installed shouldn't run during startup even without plugins.

**Expected Behavior**
I'm expecting my Discord with BetterDiscord installed to startup automatically like any other software that has that same features like Steam.

**Screenshots**
None

**Discord Version**
Stable 100054 (69f0658)
BetterDiscord 1.2.1
Host 1.0.9003 
Windows 10 64-Bit (10.0.19043)

**Additional Context**
The startup function works before I installed BetterDiscord so after I have BetterDiscord installed, that's when it happened.

**Comment by fowlis** [comment]
Check if Discord is disabled in the Startup tab in task manager.

**Comment by Waryth** [comment]
> 
> 
> Check if Discord is disabled in the Startup tab in task manager.

Discord is not listed there at all. The startup function works when I haven't installed BetterDiscord yet.

**Comment by Osyruu** [comment]
Press `Win + R` type `%localappdata%\Discord` right click to `Update.exe` --> Create shortcut then press `Win + R` again type `shell:startup `and paste the `Update.exe` shortcut in here.

**Comment by Waryth** [comment]
> 
> 
> Press `Win + R` type `%localappdata%\Discord` right click to `Update.exe` --> Create shortcut then press `Win + R` again type `shell:startup` and put the `Update.exe` shortcut in here.

I did what was written in the instructions and checked the Startup tabs in my Task Manager, to my surprised the app named "Update" got duplicated.
> 
> 
> Check if Discord is disabled in the Startup tab in task manager.

Is the app named "Update" the Discord you're talking about? I would assume so.

![Screenshot 2021-10-04 040440](https://user-images.githubusercontent.com/67438304/135769754-e8a880cd-704f-468f-a6e0-936e7ba48990.png)

The thing is it says Publisher as GitHub so at first glance it wasn't Discord. It's been Enabled with or without creating the shortcut if this is the Discord app to begin with.

I restarted my PC to test things out with the instructions of creating a shortcut of that Update.exe and no, it didn't solve the issue. All my startups apps are still working fine except Discord with BetterDiscord.

Does Discord with BetterDiscord startup works for you guys or I'm the only one experiencing this problem at the moment? I want someone to confirm the issue aside from me unless its just me.

**Comment by Osyruu** [comment]
Yes, the Update GitHub thing it's Discord, so when you launch Discord the autoupdater launcher itself.

Otherwise if it's doesn't work try uninstalling Discord and reinstalling.

**Comment by Waryth** [comment]
> 
> 
> Yes, the Update GitHub thing it's Discord, so when you launch Discord the autoupdater launch itself.
> 
> Otherwise if it's doesn't work try uninstalling Discord and reinstalling.

I did that and ran an issue along the way. BetterDiscord won't show up in the settings even when I have it installed many times as I've installed then uninstalled. This time I used repair and it finally got it installed.

Now I restarted my PC with the reinstalled Discord app along with BetterDiscord and it's finally working! Some faulty happened in Discord's side after installing BetterDiscord awhile ago. Thank you everyone for helping me resolve this issue.
